spring-jdbc/src/test/java/org/springframework/jdbc/support/KeyHolderTests.java

```diff
@@ -113,4 +113,17 @@ void getKeysWithMultipleKeyRows() {
 			.withMessageStartingWith("The getKeys method should only be used when keys for a single row are returned.");
 	}
 
+	@Test
```

Comment:
Unfortunately, I don't think it's exercising a different code path than `getKeysWithMultipleKeyRows`